.readthedocs.yaml

```diff
@@ -5,27 +5,31 @@
 # Required
 version: 2
 
-
-
 # Set the version of Python and other tools you might need
 build:
   os: ubuntu-20.04
-  apt_packages:
+  apt_packages :
     - maven
   tools:
     python: "3.9"
+  jobs:
+    pre_create_environment:
+      - echo $(pwd)
+      - rm -f src/trustyai/dep/org/trustyai/*
+      - git clone https://github.com/trustyai-explainability/trustyai-explainability.git
+      - mvn clean install -DskipTests -f trustyai-explainability/pom.xml -Pquickly -fae -e -nsu
+      - mvn clean install -DskipTests -f trustyai-explainability/explainability-arrow/pom.xml -Pshaded -fae -e -nsu
+      -  mv trustyai-explainability/explainability-arrow/target/explainability-arrow-*-SNAPSHOT.jar src/trustyai/dep/org/trustyai/
```

Comment:
I'm not an expert of ReadTheDocs but why do we need to build expl-core to build the docs? Is this build executing trustyai code?
Btw we might decide to centralise this in a single script because also ~all other CI jobs do the same steps